### DISCUSSION TITLE
Welcome to Counselling-Management-System Discussions!

### DISCUSSION THREAD

**Discussion by Moker-Sphere1822** [Moker-Sphere1822/Counselling-Management-System#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Moker-Sphere1822** [comment]
last night some files get deleted where are that

**Reply by INSHA-NISAR** [reply]
okk i ll see the issue
